synfig-studio/src/gui/duckmatic.cpp

```diff
@@ -2514,14 +2514,14 @@ Duckmatic::add_to_ducks(const synfigapp::ValueDesc& value_desc,etl::handle<Canva
 				if (tangent1_duck && tangent2_duck && !bline_point.get_split_tangent_both()) {
 					if (bline_point.get_merge_tangent_both())
 					{
-						etl::smart_ptr<synfig::Point> point(new Point(tangent1_duck->get_point()));
+						std::shared_ptr<synfig::Point> point(new Point(tangent1_duck->get_point()));
 						tangent1_duck->set_shared_point(point);
 						tangent2_duck->set_shared_point(point);
 					}
 					else
 					if (!bline_point.get_split_tangent_angle())
 					{
-						etl::smart_ptr<synfig::Angle> angle(new Angle(
+						std::shared_ptr<synfig::Angle> angle(new Angle(
```

Comment:
I suggest to replace `new` with `std::make_shared`

Reply:
Done